middlewares/usersValidation.js

```diff
@@ -0,0 +1,37 @@
+const Joi = require("joi");
+
+const schemafindUserByEmail = Joi.object({
+	email: Joi.string()
+		.email({
+			minDomainSegments: 2,
+			tlds: { allow: ["com", "net", "pl"] },
+		})
+		.required(),
+	password: Joi.string().pattern(new RegExp("^[a-zA-Z0-9]{3,30}$")).required(),
+});
+
+const schemaPatchSubscription = Joi.object({
+	subscription: Joi.string().valid("starter", "pro", "business").required(),
+});
+
+const validate = (schema, obj, next, res) => {
+	const { error } = schema.validate(obj);
+	if (error) {
+		const [{ message }] = error.details;
```

Comment:
Jestes pewien, ze `details` bedzie zawsze mialo taki ksztalt?

Reply:
tak, sprawdzałem różne błędy

Reply:
to spoczko :)